package.json

```diff
@@ -67,6 +67,7 @@
     "@types/react-transition-group": "^4.4.0",
     "@types/video.js": "^7.3.10",
     "apollo": "^2.30.2",
+    "babel-jest": "26.6.3",
```

Comment:
I have completely no clue why, but suddenly tests started crashing, saying that `babel-jest` was not found. This is super weird since running `yarn why babel-jest` outputs that we already have this package at this version because of dependency of `react-scripts`. Adding it here explicitly fixes tests though 🤔

Reply:
Interesting, let's just keep this for now then